CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift

```diff
@@ -166,23 +166,42 @@ class ErrorScreenViewController: UIViewController {
     }
 
     private func configureConstraints() {
+        configureScrollViewConstraints()
+        configureButtonsViewConstraints()
+        configureHeaderConstraints()
+        view.layoutSubviews()
+    }
+
+    private func configureHeaderConstraints() {
+        if UIDevice.current.isIpad {
+            NSLayoutConstraint.activate([
+                errorHeader.headerStack.widthAnchor.constraint(equalTo: view.widthAnchor,
+                                                               multiplier: Constants.iPadWidthMultiplier.rawValue),
+                errorHeader.headerStack.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                errorContent.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+                errorContent.widthAnchor.constraint(equalTo: view.widthAnchor,
+                                                    multiplier: Constants.iPadWidthMultiplier.rawValue)
+            ])
+        } else {
+            NSLayoutConstraint.activate([
+                errorHeader.headerStack.leadingAnchor.constraint(equalTo: view.leadingAnchor,
+                                                                 constant: Constants.iPadWidthPadding.rawValue),
+                errorHeader.headerStack.trailingAnchor.constraint(equalTo: view.trailingAnchor,
```

Comment:
[clarification]: Also I've seen that in Figma the right margin of this `errorHeader` is 24 (`iPadWidthPadding` is 35).
Maybe that's why checking this on a device is not looking very good:
<img width="564" alt="Screenshot 2023-06-23 at 13 12 33" src="https://github.com/gini/gini-mobile-ios/assets/132361182/3b164566-26ff-4370-91ef-540b17b9ca44">

In my opinion, there is too much space to the right where the content can fit but is not.

Reply:
could you please add a link to Figma where you've seen 24?

Reply:
Here. You need to select the text and press the option key with the mouse point on the right margin. 
https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?type=design&node-id=3991-53952&mode=design&t=TqYJ60xfD9Lf4pOU-4